CONTRIBUTING.md

```diff
@@ -0,0 +1,230 @@
+# Contributing Guidelines
```

Comment:
We should think about getting this file included in the generated docs site as well....but we can do that as a follow up.

Reply:
I was thinking of moving it over https://github.com/rancher-sandbox/rancher-turtles-docs/tree/main/docs folder, would that make sense?

Reply:
Yeah lets follow up